x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/ldap/support/LdapTestCase.java

```diff
@@ -107,6 +110,18 @@ public void startLdap() throws Exception {
                 ldapServer.startListening();
                 return null;
             });
+            String listenerConfig = listeners.stream()
+                .map(
+                    l -> String.format(
+                        Locale.ROOT,
+                        "(%s @ %s:%d)",
+                        l.getListenerName(),
+                        NetworkAddress.format(resolveListenAddress(l.getListenAddress())),
+                        ldapServer.getListenPort(l.getListenerName())
+                    )
+                )
+                .collect(Collectors.joining(","));
```

Comment:
Why not use `InMemoryListenerConfig#toString`? Is it because of the `listenAddress`?

Reply:
More due to the port - typically the config has port `0` and we want to know what port number was actually assigned.